optimum/neuron/distributed/encoder_decoder_models.py

```diff
@@ -111,6 +111,7 @@ def transform(
         sequence_parallel_enabled: bool = False,
         device: Optional[torch.device] = None,
         should_parallelize_layer_predicate_func: Optional[Callable[[torch.nn.Module], bool]] = None,
+        **parallel_layer_specific_kwargs,
```

Comment:
as per my experience with T5, the majority of args in `parallel_layer_specific_kwargs` sent to the ``transform()` functions in T5's parallel modules raised errors, eg:
```
TypeError: transform() got an unexpected keyword argument 'skip_linear_weight_load'
```

Reply:
I think it's just a matter of adding a missing argument here.